.github/workflows/nightly.yml

```diff
@@ -389,9 +389,47 @@ jobs:
           echo opcache.jit_hot_side_exit=1 >> /etc/php.d/opcache.ini
           echo memory_limit=-1 >> /etc/php.d/opcache.ini
           php -v
+      - name: Test PHPSeclib
+        if: always()
+        run: |
+          git clone https://github.com/phpseclib/phpseclib --branch=master
+          cd phpseclib
+          git checkout 259bd9f1e8af11726ed74acf527c2c046549061b
+          export ASAN_OPTIONS=exitcode=139
+          export PHPSECLIB_ALLOW_JIT=1
+          php /usr/bin/composer install --no-progress --ignore-platform-reqs
+          php vendor/bin/paratest --verbose --configuration=tests/phpunit.xml --runner=WrapperRunner || exit $?
```

Comment:
We're not using paratest for the other frameworks. Can we use it there too? In that case this would be best introduced in a separate PR.